lgc/interface/lgc/Pipeline.h

```diff
@@ -103,6 +103,14 @@ enum class ThreadGroupSwizzleMode : unsigned {
   Count,
 };
 
+// Enumerate the ray tracing indirect modes.
+enum class RayTracingIndirectMode : unsigned {
+  NotIndirect = 0,    // Not in indirect mode (or not ray tracing pipeline)
+  Legacy = 1,         // Legacy indirect mode
+  Continuations0 = 2, // Continuations flow that based on Continufy pass
```

Comment:
Instead of 0 and 1, maybe name this one `ContinuationsContinufy`?

Reply:
Makes sense, what about the other one, just `Continuations`?

Reply:
Yeah, I think just `Continuations` is fine.